### PR TITLE
Updated remove-dfsnroottarget #376

### PR DESCRIPTION
**Pull request by ojrb**
Updating the documentation for the switch confirm and giving example of use, this PR solved the issue #376

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: docset/windows/dfsn/remove-dfsnroottarget.md [hunk 1]
```diff
@@ -106,7 +106,9 @@ Accept wildcard characters: False
 ```
 
 ### -Confirm
-Prompts you for confirmation before running the cmdlet.
+Prompts you for confirmation before running the cmdlet.  The default value is **True** and ask for confirmation, if you do not want to confirm the operation, you must use the switch with the **False** value, as shown in this example:
```
Comment (JohanFreelancer9) [inline docset/windows/dfsn/remove-dfsnroottarget.md]:
```suggestion
Prompts you for confirmation before running the cmdlet.  The default value is **True** and ask for confirmation. If you do not want to confirm the operation, you must use the switch with the **False** value, as shown in this example:
```

Comment (JohanFreelancer9) [inline docset/windows/dfsn/remove-dfsnroottarget.md]:
```suggestion
Prompts you for confirmation before running the cmdlet.  The default value is **True** and asks for confirmation. If you do not want to confirm the operation, you must use the switch with the **False** value, as shown in this example:
```

**Comment by ojrb** [comment]
Hi @e0i  I accepted the copy review suggestions !

**Pull request merged**